datasync/chaindatafetcher/kas/repository.go

```diff
@@ -88,3 +97,54 @@ func (r *repository) SetComponent(component interface{}) {
 		r.blockchainApi = c
 	}
 }
+
+func makeEOAListStr(eoaList map[common.Address]struct{}) string {
+	eoaStrList := ""
+	for key := range eoaList {
+		eoaStrList += "\""
+		eoaStrList += strings.ToLower(key.String())
+		eoaStrList += "\""
+		eoaStrList += ","
+	}
+	return eoaStrList[:len(eoaStrList)-1]
+}
+
+func (r *repository) InvalidateCacheEOAList(eoaList map[common.Address]struct{}) {
+	if len(eoaList) <= 0 || !r.config.CacheUse {
+		return
+	}
+
+	url := r.config.CacheInvalidationURL
+	method := "POST"
+	payloadStr := fmt.Sprintf(`{"type": "stateChange","payload": {"addresses": [%v]}}`, makeEOAListStr(eoaList))
+	payload := strings.NewReader(payloadStr)
+
+	// set up timeout for API call
+	ctx, cancel := context.WithTimeout(context.Background(), cacheTimeout)
+	defer cancel()
+
+	req, err := http.NewRequestWithContext(ctx, method, url, payload)
```

Comment:
This is the only place where the `method` is used and is always set as "POST".
Is there is no possibility to change the http request method or reuse the variable, how about using it as a literal value? That would reduce some complexity.

Reply:
Applied. 9d5a93976cec83e2f3d0f29f0ebdc67a91a37327